libs/components/popovers/src/lib/modules/dropdown/dropdown.component.html

```diff
@@ -1,4 +1,10 @@
 <div class="sky-dropdown">
+  <div class="sky-screen-reader-only" id="sky-sr-label-context-menu">
```

Comment:
Same issue here with the hardcoded ID. Since you're not referencing it in your component, you can use the `skyId` directive instead of the service and then reference the ID elsewhere in the template.

```
<div class="sky-screen-reader-only" skyId #srLabelContextMenu="skyId">
...
<button [attr.aria-labelledby]="srLabelContextMenu.id">
```